great_expectations/data_context/migrator/cloud_migrator.py

```diff
@@ -297,16 +299,14 @@ def _send_validation_results(
         serialized_validation_results: Dict[str, dict],
         test_migrate: bool,
     ) -> None:
-        print("[Step 4/4]: Sending validation results]")
-        if test_migrate:
-            return
```

Comment:
No need to make this change in this PR (or even at all).
But just wanted to float the idea of using a generator for running all these steps.

Very rough idea. This might not be appropriate at all.

```python
def migration_pipeline_generator() -> Generator[str, None, None]:
    # step_one stuff
    yield "step one message"
   # step two stuff
   yield "step two message"
```

And then you could iterate through it at once or if need be you can pass it around and iterate through the steps one by one. The generator object preserves its place.

```python
for step, message in enumerate(migration_pipeline_generator(), start=1):
  print(f"[Step {step}]: {message}")
```

```python
migration_pipeline = migration_pipeline_generator()

step1_result = next(migration_pipline)

some_other_function(migration_pipeline)
```